### DISCUSSION TITLE
"Running with systemd-docker" wiki page does not use systemd-docker

### DISCUSSION THREAD

**Discussion by sr1canskhsia** [dani-garcia/vaultwarden#1947] [Q&A]
I was following [this wiki page](https://github.com/dani-garcia/vaultwarden/wiki/Running-with-systemd-docker) to have systemd managing my vaultwarden instance. It is stated that we need to install `systemd-docker` package, yet the actual unit file does not use it at all. Is this intentional? If we actually do not need the `systemd-docker` package, can we update this wiki page to remove all the `systemd-docker` references? Otherwise it is just extremely confusing

**Comment by BlackDex** [comment]
It's a wiki, go ahead and improve it.
